rules-reviewed/eap7/eap71/tests/hibernate51-53.windup.test.xml

```diff
@@ -160,20 +160,16 @@
                 </perform>
             </rule>
             -->
-            <!--https://issues.jboss.org/browse/WINDUPRULE-387
             <rule id="hibernate51-53-01100-test">
                 <when>
                     <not>
-                        <iterable-filter size="5">
-                            <hint-exists message="" />
-                        </iterable-filter>
+                        <hint-exists message="Existing applications migrating to 5.3 and using the `@TableGenerator`" />
```

Comment:
Better keep the check on the number of instances
```suggestion
                        <iterable-filter size="1">
                            <hint-exists message="Existing applications migrating to 5.3 and using the `@TableGenerator`" />
                        </iterable-filter>
```

Reply:
+1